### DISCUSSION TITLE
persistent scrachpad

### DISCUSSION THREAD

**Discussion by tokisuno** [qtile/qtile#4306] [Ideas]
i don't know if this is a feature or not, but being able to keep a scratchpad up until i hit the toggle key again would be useful. sometimes my cursor moves outside of the scratchpad so i have to re enable it and it gets annoying.

**Comment by ervinpopescu** [comment]
Set `on_focus_lost_hide=False` on the Dropdown you want to keep up. 
https://qtile.readthedocs.io/en/stable/manual/config/groups.html#dropdown
